templates/minecraft-server/meta.yaml

```
@@ -0,0 +1,52 @@
+name: Minecraft Server
+description: Minecraft Server that will automatically download the latest stable version at startup. You can also run/upgrade to any specific version or the latest snapshot.
+instructions: null
+changeLog:
+  - date: 2023-03-18
+    description: First Release
+links:
+  - label: Github
+    url: https://github.com/itzg/docker-minecraft-server
+  - label: Docker
+    url: https://hub.docker.com/r/itzg/minecraft-server
+contributors:
+  - name: BerkSMBL
+    url: https://berksmbl.com
+schema:
+  type: object
+  required:
+    - projectName
+    - appServiceName
+    - appServiceImage
+    - serverPort
+  properties:
+    projectName:
+      type: string
+      title: Project Name
+    appServiceName:
+      type: string
+      title: App Service Name
+      default: minecraft-server
+    appServiceImage:
+      type: string
+      title: App Service Image
+      default: itzg/minecraft-server:2023.2.0
+    eula:
+      type: boolean
+      title: Please accept the Minecraft EULA
+      default: true
```

Since it's a EULA, i'd recommend to have it on `false` by default and make it `required`.
```suggestion
      default: false
```

---

If they do not accept this, the installation does not start.

Since I prepared it with the logic of 1 Click install, I want to have a ready-made template when the "Create" button is clicked directly without entering any value.

---

Even if I mark it as "required" because the boolean values are false and true, it returns a value if they don't accept it.